### DISCUSSION TITLE
Bytes masking in Micropython

### DISCUSSION THREAD

**Discussion by labodezao** [micropython/micropython#10300] [Using MicroPython] [answered]
Hi,
I have a while instance : 
while (( self.bus.readfrom_mem(0x6d, 0x06, 1) & 0x08)> 0) :
Which make this error : TypeError: unsupported types for __and__: 'bytes', 'int'

That I could not solve because in mycropython documentation, readfrom_mem() is supposed to return a Bytes type...

Thanks by advance

**Comment by robert-hh** [comment, marked as answer]
try;
`while (( self.bus.readfrom_mem(0x6d, 0x06, 1)[0] & 0x08)> 0) :`

**Reply by labodezao** [reply]
thanks, it works
